internal/service/cloudbackupschedule/resource_cloud_backup_schedule_test.go

```diff
@@ -723,106 +719,95 @@ func configAdvancedPolicies(info *acc.ClusterInfo, p *admin.DiskBackupSnapshotSc
-				project_id       = %[2]s
-
-				auto_export_enabled      = true
-				reference_hour_of_day    = 20
-				reference_minute_of_hour = "05"
-				restore_window_days      = 4
-
-				policy_item_daily {
-				frequency_interval = 1
-				retention_unit     = "days"
-				retention_value    = 4
-				}
-				export {
-					export_bucket_id = mongodbatlas_cloud_backup_snapshot_export_bucket.test.export_bucket_id
-					frequency_type   = "daily"
-				}
-			}
-
-			resource "aws_s3_bucket" "backup" {
-				bucket = "${local.mongodbatlas_project_id}-s3-mongodb-backups"
-				force_destroy = true
-					object_lock_configuration {
-						object_lock_enabled = "Enabled"
-					}
-			}
-
-			resource "mongodbatlas_cloud_provider_access_setup" "setup_only" {
-				project_id       = %[2]s
-				provider_name = "AWS"
-			}
-
-			resource "mongodbatlas_cloud_provider_access_authorization" "auth_role" {
-				project_id       = %[2]s
-				role_id    = mongodbatlas_cloud_provider_access_setup.setup_only.role_id
-
-				aws {
-					iam_assumed_role_arn = aws_iam_role.test_role.arn
-				}
-			}
-
-			resource "mongodbatlas_cloud_backup_snapshot_export_bucket" "test" {
-				project_id       = %[2]s
-
-				iam_role_id    = mongodbatlas_cloud_provider_access_authorization.auth_role.role_id
-				bucket_name    = aws_s3_bucket.backup.bucket
-				cloud_provider = "AWS"
-			}
-
-			resource "aws_iam_role_policy" "test_policy" {
-				name = %[3]q
-				role = aws_iam_role.test_role.id
-
-				policy = <<-EOF
-				{
-					"Version": "2012-10-17",
-					"Statement": [
-					{
-						"Effect": "Deny",
-						"Action": "*",
-						"Resource": "*"
-					}
-					]
-				}
-				EOF
-			}
-
-			resource "aws_iam_role" "test_role" {
-				name = %[4]q
-
-				assume_role_policy = <<EOF
-			{
-				"Version": "2012-10-17",
-				"Statement": [
-					{
-						"Effect": "Allow",
-						"Principal": {
-							"AWS": "${mongodbatlas_cloud_provider_access_setup.setup_only.aws_config.0.atlas_aws_account_arn}"
-						},
-						"Action": "sts:AssumeRole",
-						"Condition": {
-							"StringEquals": {
-								"sts:ExternalId": "${mongodbatlas_cloud_provider_access_setup.setup_only.aws_config.0.atlas_assumed_role_external_id}"
-							}
-						}
-					}
-				]
-			}
-			EOF
-		}
-	`, info.ClusterNameStr, info.ProjectIDStr, policyName, roleName, awsAccessKey, awsSecretKey, region)
+    resource "mongodbatlas_cloud_backup_schedule" "schedule_test" {
+        cluster_name     = %[1]s
+        project_id       = %[2]s
+        auto_export_enabled      = true
+        reference_hour_of_day    = 20
+        reference_minute_of_hour = "05"
+        restore_window_days      = 4
+        policy_item_daily {
+            frequency_interval = 1
+            retention_unit     = "days"
+            retention_value    = 4
+        }
+        export {
+            export_bucket_id = mongodbatlas_cloud_backup_snapshot_export_bucket.test.export_bucket_id
+            frequency_type   = "monthly"
```

Comment:
[nit] intentional change from `daily` -> `monthly`?

Reply:
Yup, I effectively had to change the configuration because it was wrong. I used this one that I was familiar with. Also, not all clusters by default allow daily export (I learned this recently)